modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/fragment/renderer/FDSAdminFragmentRenderer.java

```diff
@@ -1006,6 +1006,16 @@ private JSONArray _getSortsJSONArray(
 			(ObjectEntry objectEntry) -> {
 				Map<String, Object> properties = objectEntry.getProperties();
 
+				String label = (String)properties.get("label");
+
+				if (label.equals(StringPool.BLANK)) {
+					Map<String, String> labelI18n =
+						(Map<String, String>)properties.get("label_i18n");
+
+					label = labelI18n.get(
+						LocaleUtil.toLanguageId(LocaleUtil.getSiteDefault()));
```

Comment:
This one uses the site default since the label is already using `LocaleUtil.getMostRelevantLocale()` when getting the object entry in `_getRelatedObjectEntries`. So when `label` is blank, it's assuming that the relevant locale's label is not defined and fallbacks to the site default.

https://github.com/liferay-frontend/liferay-portal/blob/1daa263cbb78f6c5be218264f4673547a7efddb1/modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/fragment/renderer/FDSAdminFragmentRenderer.java#L919-L922

Reply:
I see, makes sense, thanks!